### DISCUSSION TITLE
'send-abort-event' key binding to "c-]"

### DISCUSSION THREAD

**Discussion by alanz** [lem-project/lem#1523] [General]
I am trying to set up a familiar `evil-mode` environment for myself in lem, and attempted

```lisp
(define-key lem-vi-mode:*normal-keymap* "c-]" 'lem/language-mode:find-definitions)
``` 

To my surprise, pressing that key combination pops up a stack trace instead.  After some spelunking, I discovered that `c-]` is hard coded (in sdl2 at least) to call `send-abort-event`.

See https://github.com/lem-project/lem/blob/0de0da6282081f176d44ef44326382b594d1b36b/frontends/sdl2/keyboard.lisp#L124-L127

Is there a reason for this particular key choice?  
How does this abort functionality differ from the emacs `c-g` universal abort behaviour?
Should it be bound to `c-g` instead?

**Comment by g-gundam** [comment]
FYI, there is something bound to `C-g` already.

| keymap | function |
| --- | --- |
| `lem-vi-mode:*motion-keymap*` | `lem-vi-mode/commands:vi-keyboard-quit` |
| `lem-core:*global-keymap*` | `lem-core/commands/other:keyboard-quit` |
